CHANGELOG.md

```diff
@@ -1 +1,15 @@
+<!-- markdownlint-disable MD024 -->
```

Comment:
Why is this needed?

Reply:
This rule is for `MD024/no-duplicate-heading`.  I could have sworn that I copied/pasted this from the Functions CHANGELOG and I thought it made sense since we have a lot of headings that are duplicated (Added, Removed, etc.) but I can't find it anymore... So I guess I'll get rid of it.